lighty-modules/lighty-restconf-nb-community/src/main/java/io/lighty/modules/northbound/restconf/community/impl/CommunityRestConf.java

```diff
@@ -84,12 +86,11 @@ protected boolean initProcedure() {
 
         this.schemaCtxHandler
                 = new SchemaContextHandler(this.domDataBroker, this.domSchemaService);
-        this.schemaCtxHandler.init();
```

Comment:
We can remove `schemaContextHandler` completely from this class.

Reply:
removed in 7e95206af83546f98cd62b0d592c8b4705566654

Reply:
basically removing this code causes restconf to break: 
`new SchemaContextHandler(this.domDataBroker, this.domSchemaService);`